### ISSUE TITLE
Separation of upcoming events

### ISSUE THREAD

**Issue by SimonHettrick** [softwaresaved/carpentry-data-analysis#3]
Just had a look at the carpentry events that are plotted here:
https://35.177.39.55/training/carpentry-events-uk
And they look great!

It occurs to me that it would be really useful to have another map showing only the upcoming events (after all, one of our key markets will be "people who want to attend SWC but don't know where a workshop is held").

Would it be possible to write another script, or add to the current script, so that it produces a csv with only the upcoming events?

**Comment by anenadic** [comment]
Yes, that should not be a problem. I think that is where the idea came from originally - plot the upcoming ones on the website (and a separate list of past workshops). Will talk to @priyankasingh and @steve-crouch.

**Comment by priyankasingh** [comment]
Hi Aleks, 

Steve and I had a discussion about it and Mario gave his suggestions too. We think it'll be good if there are 3 different CSV files.

1) CSV file with all the workshops (past, present and future events): This should be in chronological order. So the future events are at the bottom, so when we plot it on the map, the latest workshops are plotted at the end and shows on top of the map.

2) CSV file with past workshops: This should be in reverse chronological order, so most recent one are on top of the list to view.

3) CSV file with future workshops: This should be in chronological order so we can view the future events.

Let me know if you have any questions.

Thanks and regards,
Priyanka

**Comment by anenadic** [comment]
I do not see a problem with that.
